src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -74,6 +75,7 @@ case class BenchmarkResult(
     planningTime: Option[Double] = None,
     executionTime: Option[Double] = None,
     result: Option[Long] = None,
+    doubleResults: Map[String, Double] = Map.empty,
```

Comment:
Streaming perf jobs need to collect multiple results and may add more later. And we will encounter 22 fields issue if just adding all results to different fields.